extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/ItagItem.java

```diff
@@ -3,17 +3,24 @@
 import org.schabi.newpipe.extractor.MediaFormat;
 import org.schabi.newpipe.extractor.exceptions.ParsingException;
 
+import javax.annotation.Nonnull;
+
 import static org.schabi.newpipe.extractor.MediaFormat.*;
 import static org.schabi.newpipe.extractor.services.youtube.ItagItem.ItagType.*;
 
-public class ItagItem {
+import java.io.Serializable;
+
+public class ItagItem implements Serializable {
+    private static final long serialVersionUID = 2885399300064582464L;
```

Comment:
Is there a reason why we need ``serialVersionUID`` now? (applies for more classes)

Reply:
I just applied what the [Serializable interface](https://docs.oracle.com/javase/8/docs/api/java/io/Serializable.html) recommends:

> However, it is strongly recommended that all serializable classes other than enum types explicitly declare serialVersionUID values, since the default serialVersionUID computation is highly sensitive to class details that may vary depending on compiler implementations, and can thus result in unexpected `InvalidClassExceptions` during deserialization. Therefore, to guarantee a consistent serialVersionUID value across different java compiler implementations, a serializable class must declare an explicit serialVersionUID value.

Reply:
This does not matter for NewPipe's use of the extractor. However, we do not know what others do with it. Should be good.

Reply:
> This does not matter for NewPipe's use of the extractor.

👍 

> However, we do not know what others do with it. Should be good.

My problem with this:
* It's completely unrelated to the PR/issue
* There are now a handful of classes that have this and the most others do not
* You don't use the java built in serialization - it caused so many [massive security holes](https://snyk.io/blog/serialization-and-deserialization-in-java/) in the past. I see no reason why we should support it in the first place.
* Why should we support/maintain something we don't use? How can we even assure that it's correctly implemented?